src/main/java/hello/HelloWorldHttpController.java

```diff
@@ -16,6 +16,8 @@ public String greeting(@RequestParam(value="name", required=false, defaultValue=
         if (true){
             int r = 4 + 5;
             System.out.print(r);
+            String token2 = new Application().generateCSRFToken();
+            System.out.print(token2);
         }
         else {
             int r =1;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=squid%3AIndentationCheck)